### ISSUE TITLE
Slab Reverse Recipes Overwrite Vanilla

### ISSUE THREAD

**Issue by Darkosto** [mrh0/buildersaddition#31]
Hello!

I've discovered a number of recipes are overriding vanilla recipes:

Chiseled Quartz, Chiseled Stone bricks, & Red Sandstone Slab are the ones I could find so far.

Is there a possibility of allowing both vanilla and modded recipes to coexist? 

Mod Version: 1.16.5-20210407a
Forge: 36.1.0
MC: 1.16.5

Thank you!

**Comment by mrh0** [comment]
You are right, never thought of this, I will have to find a solution to this.

**Comment by mrh0** [comment]
fixed
